src/Modules/Microsoft.Azure.Functions.PowerShellWorker/Microsoft.Azure.Functions.PowerShellWorker.psm1

```diff
@@ -0,0 +1,110 @@
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+function CheckIfDurableFunctionsEnabled {
+	if (-not [bool]$env:PSWorkerEnableExperimentalDurableFunctions) {
+		throw 'Durable function is not yet supported for PowerShell.'
```

Comment:
Could you please make this a error record? E.g.,

```
function ThrowTerminatingError
{
    param 
    (
        [Parameter(Mandatory=$true)]
        [ValidateNotNullOrEmpty()]
        [System.String]
        $ErrorId,

        [Parameter(Mandatory=$true)]
        [ValidateNotNullOrEmpty()]
        [System.String]
        $ErrorMessage,

        [Parameter(Mandatory=$true)]
        [ValidateNotNullOrEmpty()]
        [System.Management.Automation.ErrorCategory]
        $ErrorCategory,

        [Exception]
        $Exception,

        [object]
        $TargetObject
    )

    if (-not $Exception)
    {
        $Exception = New-Object -TypeName System.Exception -ArgumentList $ErrorMessage
    }

    $errorRecord = New-Object -TypeName System.Management.Automation.ErrorRecord -ArgumentList ($Exception, $ErrorId, $ErrorCategory, $TargetObject)
    throw $errorRecord
}
```

Reply:
Why? Is this to make it possible to specify ErrorId, ErrorCategory, and TargetObject? I would agree this may be useful in some cases. But let's take a step back and look at the actual use case: we would add all this code just so that someone who tries to use durable functions (before we promised that they would work!) can get nicer ErrorId and ErrorCategory... Comparing this amount of code with the achieved goal, this looks like an overkill to me.

Reply:
I agree. Let it as is.